packages/zowe-explorer/src/trees/dataset/DatasetFSProvider.ts

```diff
@@ -588,10 +588,18 @@ export class DatasetFSProvider extends BaseProvider implements vscode.FileSystem
         try {
             const mvsApi = ZoweExplorerApiRegister.getMvsApi(entry.metadata.profile);
             const profile = Profiles.getInstance().loadNamedProfile(entry.metadata.profile.name);
-            const profileEncoding = entry.encoding ? null : profile.profile?.encoding; // use profile encoding rather than metadata encoding
+            const profileEncoding = entry.encoding ? null : profile.profile?.encoding;
+
+            let binary = false;
+            if (encoding === "binary") {
+                binary = true;
+            } else if (entry.encoding?.kind === "binary") {
+                binary = true;
+            }
```

Comment:
We should use `const` when possible, this is a good use case for it:

```suggestion
            const binary = encoding === "binary" || entry.encoding?.kind === "binary";
```